docs/DeveloperGuide.md

```diff
@@ -1,38 +1,219 @@
+- **Robust Data Protection**: Say goodbye to the vulnerabilities of paper records! Our digital solution offers enhanced data protection, ensuring your records remain safe from damage, loss, or unauthorized access. With advanced encryption and backup mechanisms in place, your data's safety is our priority.
+- **Eco-Friendly**: Transition from paper to digital and contribute to a greener planet. Not only does this transition reduce clutter and the risk of loss, but it also significantly diminishes your carbon footprint.
+- **Cost-Effective**: Eliminate the recurring costs of paper, printing, and storage. With our solution, you'll experience a noticeable reduction in operational expenses while benefiting from advanced data management capabilities.
+
+[⬆ Back to top](https://ay2324s1-cs2113t-w11-2.github.io/tp/DeveloperGuide.html#table-of-contents)
+
+--------------------------------------------------------------------------------------------------------------------------------------
 
 ## User Stories
 
 |Version| As a ... | I want to ... | So that I can ...|
 |--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
+|v1.0|new user|see help instructions|refer to them when I forget how to use the application|
+|v1.0|zoo clerk|add an animal entry|record the animals in the zoo, and refer to them afterwards|
+|v1.0|zoo clerk|delete an animal entry|remove redundant or invalid animal entry|
+|v1.0|zoo clerk|list all the entries|see what entries I have entered previously, and refer to them|
 |v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
 
+[⬆ Back to top](https://ay2324s1-cs2113t-w11-2.github.io/tp/DeveloperGuide.html#table-of-contents)
+
+--------------------------------------------------------------------------------------------------------------------------------------
+
+
+
 ## Non-Functional Requirements
 
 {Give non-functional requirements}
 
+[⬆ Back to top](https://ay2324s1-cs2113t-w11-2.github.io/tp/DeveloperGuide.html#table-of-contents)
+
+--------------------------------------------------------------------------------------------------------------------------------------
+
+
+## Manual Testing
+
+{Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+[⬆ Back to top](https://ay2324s1-cs2113t-w11-2.github.io/tp/DeveloperGuide.html#table-of-contents)
+
+--------------------------------------------------------------------------------------------------------------------------------------
+
+## Command Summary
+
+[⬆ Back to top](https://ay2324s1-cs2113t-w11-2.github.io/tp/DeveloperGuide.html#table-of-contents)
+
+--------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
Overall DG is incomplete, well formatted and easy to understand but insufficient details and diagrams